mambaSharedFramework/HLSInterstitialValueTypes.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  HLSInterstitialValueTypes.swift
+//  mamba
+//
+//  Created by Migneco, Ray on 10/22/24.
+//
+
+import Foundation
+
+/// specifies how the client should align interstitial content to the primary content
+public struct HLSInterstitialAlignment: FailableStringLiteralConvertible, Equatable {
+
+    public enum Snap: String, CaseIterable {
+        /// client SHOULD locate the segment boundary closest to the scheduled resumption point from the
+        /// interstitial in the Media Playlist of the primary content and resume playback of primary content at that boundary.
+        case snapIn = "IN"
+        
+        /// client SHOULD locate the segment boundary closest to the START-DATE of the interstitial in the
+        /// Media Playlist of the primary content and transition to the interstitial at that boundary.
+        case snapOut = "OUT"
```

Comment:
Total nit-pick, but `snap` is already in the enum name, so feels like `.in` and `.out` are better case names (as well as being more terse):
```suggestion
        /// client SHOULD locate the segment boundary closest to the scheduled resumption point from the
        /// interstitial in the Media Playlist of the primary content and resume playback of primary content at that boundary.
        case in = "IN"
        
        /// client SHOULD locate the segment boundary closest to the START-DATE of the interstitial in the
        /// Media Playlist of the primary content and transition to the interstitial at that boundary.
        case out = "OUT"
```
But really, I don't mind that much... Just calling it out in case it was an oversight.

Reply:
No that's good feedback, will update